### PR TITLE
Redesign/navbar

### PR DESCRIPTION
**Pull request by mistikk**

### CODE REVIEW

File: src/components/bottomTabBar/view/tabbar.js [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+    justifyContent: 'space-between',
+  },
+  navItem: {
+    alignItems: 'center',
+    zIndex: 0,
+    padding: 20,
+    width: (deviceWidth - 38) / 5,
+  },
+  circle: {
+    bottom: 25,
```
Comment (ue) [inline src/components/bottomTabBar/view/tabbar.js]: move to style file

File: src/components/bottomTabBar/view/bottomTabBarView.js [hunk 2]
```diff
@@ -70,26 +71,32 @@ class BottomTabBarView extends PureComponent {
...
-            >
-              <TouchableOpacity onPress={() => this._jumpTo(route)}>
-                {renderIcon({
-                  route,
-                  focused: index === idx,
-                  tintColor: index === idx ? activeTintColor : inactiveTintColor,
-                })}
-              </TouchableOpacity>
-            </View>
+      <SafeAreaView style={styles.wrapper}>
```
Comment (ue) [inline src/components/bottomTabBar/view/bottomTabBarView.js]: can be function comp

**Comment by mistikk** [comment]
@ue updated

File: src/components/bottomTabBar/view/bottomTabBarStyles.js [hunk 3]
```diff
@@ -1,16 +1,31 @@
...
-    height: 56,
     width: '$deviceWidth',
-    backgroundColor: '$primaryBackgroundColor',
-    borderTopWidth: 0.1,
-    shadowOpacity: 0.1,
-    elevation: 10,
+    position: 'absolute',
+    bottom: 0,
+    left: 0,
+    zIndex: 99999,
```
Comment (ue) [inline src/components/bottomTabBar/view/bottomTabBarStyles.js]: use max 1000

**Pull request merged**